### ISSUE TITLE
It seems multiple params are not parsed shown in intellisense in the JS-DOC style annotations

### ISSUE THREAD

**Issue by weisspure** [thqby/vscode-autohotkey2-lsp#539]
as title suggests
![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/26007794/e6a821ba-e6cc-4cf8-a47e-d351f4b85bb9)

**Comment by thqby** [comment]
Shows only the current parameters, is there a problem?

**Comment by weisspure** [comment]
I think I see what you mean, this __New method had 3 params, and I thought regardless of whether I was entering the first param, 2nd or 3rd it would only show the first param tag of the class method on the hint despite the fact 3 params are defined in the signature and in the comment block above the method signature.

I tried recreating the issue and it appears after I type the first comma the second param does indeed appear (I'm guessing because it's optional?)

But I did notice that if I backspace after immediately adding the comma that it goes back to one param hint and adding the comma doesn't bring the second back. I think I organically walked into this issue backspacing when I realised I entered the first param wrong, backspaced to get back to it and was expecting the 2nd param tag to appear.

This may be what lead me to believe I couldn't get the second hint working.

![image](https://github.com/thqby/vscode-autohotkey2-lsp/assets/26007794/5dff4d9e-afd6-49ff-ab01-e56b61e2d31f)

If there's a good reason for the second param to not appear the 2nd time you're inputing it after deleting a mistake I'll be happy to close my issue, otherwise I think that might be worth fixing?

By the way, I don't intend for this to be just a complaint, I love your extension, thanks for the work you've done already. I tried 2 more popular extensions but they barely worked by comparison: this is the only problem I've ran into on your extension.
